setup.py

```diff
@@ -36,6 +36,8 @@
         'future',
         'gevent',
         'gunicorn',
+        'gym<=0.9.5',  # gym in version 0.9.6 has some temporary issues.
```

Comment:
See #520 and the discussion whether `gym` should be a hard dependency, or just a soft one.

Reply:
With this PR we are proposing a Reinforcement Learning module to t2t for which gym is required.
I don't see a clear way to keep gym a "soft dependency" and to include rl into Travis test suite, which should be done.

I understand that broken version 0.9.6 of gym is a bit problematic, but I don't expect this kind of problems to reappear too often. Specifying a single, working version of gym and bumping it from time to time doesn't seem like too much of a hustle.

Reply:
OK. Personally, I don't mind the extra dependency. I agree that the ability to test your RL code with Travis changes the game.
The alternative would be to add `pip install gym==0.9.5` to https://github.com/tensorflow/tensor2tensor/blob/103d0571ef48d82b61aae67cd61d716738f9fcfe/.travis.yml#L8-L13
I just wanted to link the discussion, where @epurdyf, @cbockman and @rsepassi were involved.